app/css/custom.scss

```diff
@@ -871,4 +947,4 @@ blockquote {
       }
     }
   }
-}
+}
```

Comment:
Yes there is a structure. But it was never used. It was supposed to be used with main.scss but our main sass file is the custom.scss which was imported in the Zhishi.react.js but main.scss was never used. Making any thing we do in the app/css/4-pages kind of un-reflective. Do check and get back to me on this. I tried it before moving the codes to custom.scss. I've updated to PR to include an import of just user_profile page